crates/rattler_networking/Cargo.toml

```
@@ -11,10 +11,11 @@ license.workspace = true
 readme.workspace = true
 
 [features]
-default = ["native-tls"]
+default = ["native-tls", "s3"]
```

```suggestion
default = ["native-tls"]
```

probably this makes more sense?